examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/ImageViewer.common.kt

```diff
@@ -62,8 +73,8 @@ internal fun ImageViewerWithProvidedDependencies(
                 GalleryScreen(
                     pictures = pictures,
                     selectedPictureIndex = selectedPictureIndex,
-                    onClickPreviewPicture = { previewPictureId ->
-                        navigationStack.push(MemoryPage(mutableStateOf(previewPictureId)))
+                    onClickPreviewPicture = { previewPictureIndex ->
```

Comment:
how does it work? Is it previewPictureIndex or picture?

Reply:
Oooops, wrong name. I want to make it as previewPictureIndex: Int
But for now it have type previewPictureIndex: Picture

I will change it to index

Reply:
Fixed to previewPictureIndex: Int